percona-packaging/redhat/percona-server-mongodb.spec.template

```diff
@@ -166,13 +167,13 @@ cp -r $RPM_BUILD_DIR/%{src_dir}/mongo-tools ${GOPATH}/src/github.com/mongodb
 pushd ${GOPATH}/src/github.com/mongodb/mongo-tools
 . ./set_tools_revision.sh
 sed -i '12d' buildscript/build.go
-sed -i '169,178d' buildscript/build.go
+sed -i '167,176d' buildscript/build.go
 sed -i "s:versionStr,:\"$PSMDB_TOOLS_REVISION\",:" buildscript/build.go
 sed -i "s:gitCommit):\"$PSMDB_TOOLS_COMMIT_HASH\"):" buildscript/build.go
 ./make build
 mv bin/* $RPM_BUILD_DIR/%{src_dir}/bin
 popd
-popd
```

Comment:
it should be here as we make 2 pushd and only 1 popd is left in this case

Reply:
This part is cherry-pickked from commit 303f31d69ffb in the v4.4 branch. Looks like the same problem exists in v4.4